stock_getter/frontend_stockgetter/scripts/card.js

```diff
@@ -92,6 +92,8 @@ class Card {
         console.log(`Getting data for stock symbol ${this.stockSymbol} 
             at ${this.grain} grain`)
 
+        this.forecast = forecast;

```

Comment:
Fix for Chart having forecast items in the legends when forecast is not selected